unity/Holo/Assets/SharedExperience/Scripts/BulletController.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This file seems like it is not necessary in our project :) Unless "bullet" here is something more technological (some mechanism for synchronization?) instead of game-like bullets.